core/create/src/prompts/systemCode.ts

```diff
@@ -0,0 +1,38 @@
+import { styles } from '@dotcom-tool-kit/logger'
+import prompt from 'prompts'
+import type { PackageJson } from 'type-fest'
+import { BizOpsSystem, getBizOpsSystem } from '../bizOps'
+
+export interface ConfirmationParams {
+  packageJson: PackageJson
+}
+
+export default async ({ packageJson }: ConfirmationParams): Promise<BizOpsSystem | undefined> => {
+  const guessedSystemCode = packageJson.name
+  let bizOpsSystem
+  try {
+    if (!guessedSystemCode) {
+      // fall back to the prompt if we don't have a system code to guess
+      throw new Error()
```

Comment:
**question:** is this error purposely empty?

Reply:
Ahh to force the catch.

Reply:
yep, I couldn't think of a nicer way to handle this with JS's control flow constructs